moveit_py/src/moveit_py/pybind11_utils/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+add_library(pybind11_utils SHARED
+  src/copy_ros_msg.cpp
+)
+
+target_include_directories(pybind11_utils PUBLIC
```

Comment:
I think, this can be simplified to: 
`target_include_directories(pybind11_utils PUBLIC include)`

... as the (relative) include path is the same in both cases.